docs/book/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+Giving feedback to a user is an important part of a good application.
+Flash messages are used to notify the user about a successful form submit, failure on saving in the database, wrong authentification credentials or something similar.
+At the end of a request, a message is created and shown to the user at the next request.
+The flash messages are self-expiring and session-based.
+
+To create and expose flash messages in a laminas-mvc-based application this packages provides:
```

Comment:
Sorry to be nit-picky, but I think the sentence is missing a comma before "this".